archunit/src/main/java/com/tngtech/archunit/lang/conditions/ArchConditions.java

```diff
@@ -928,91 +928,156 @@ public static AllDependenciesCondition onlyHaveDependenciesWhere(DescribedPredic
     @PublicAPI(usage = ACCESS)
     public static ArchCondition<JavaClass> notBeInterfaces() {
         return not(InterfacesCondition.BE_INTERFACES);
     }
 
+    /**
+     * @return A condition matching classes that are enums as defined in {@link JavaClass#isEnum()}
+     * @see #notBeEnums()
+     */
     @PublicAPI(usage = ACCESS)
     public static ArchCondition<JavaClass> beEnums() {
         return EnumsCondition.BE_ENUMS;
     }
 
+    /**
+     * @return negation of {@link #beEnums()}
+     */
     @PublicAPI(usage = ACCESS)
     public static ArchCondition<JavaClass> notBeEnums() {
         return not(EnumsCondition.BE_ENUMS);
     }
 
+    /**
+     * @return A condition matching classes that are records according to the Java Language Specification
+     * as defined in {@link JavaClass#isRecord()}
+     * @see #notBeRecords()
+     */
     @PublicAPI(usage = ACCESS)
     public static ArchCondition<JavaClass> beRecords() {
         return RecordsCondition.BE_RECORDS;
     }
 
+    /**
+     * @return negation of {@link #beRecords()}
+     */
     @PublicAPI(usage = ACCESS)
     public static ArchCondition<JavaClass> notBeRecords() {
         return not(RecordsCondition.BE_RECORDS);
     }
 
+    /**
+     * @return A condition matching classes that are top level classes (not nested classes)
+     * as defined in {@link JavaClass#isTopLevelClass()}
+     * @see #notBeTopLevelClasses()
+     */
     @PublicAPI(usage = ACCESS)
     public static ArchCondition<JavaClass> beTopLevelClasses() {
         return BE_TOP_LEVEL_CLASSES;
     }
 
+    /**
+     * @return negation of {@link #beTopLevelClasses()}
+     */
     @PublicAPI(usage = ACCESS)
     public static ArchCondition<JavaClass> notBeTopLevelClasses() {
         return not(BE_TOP_LEVEL_CLASSES);
     }
 
+    /**
+     * @return A condition matching classes that are nested classes as defined in {@link JavaClass#isNestedClass()}
+     * @see #notBeNestedClasses()
+     */
     @PublicAPI(usage = ACCESS)
     public static ArchCondition<JavaClass> beNestedClasses() {
         return BE_NESTED_CLASSES;
     }
 
+    /**
+     * @return negation of {@link #beNestedClasses()}
+     */
     @PublicAPI(usage = ACCESS)
     public static ArchCondition<JavaClass> notBeNestedClasses() {
         return not(BE_NESTED_CLASSES);
     }
 
+    /**
+     * @return A condition matching classes that are member classes as defined in {@link JavaClass#isMemberClass()}
+     * @see #notBeMemberClasses()
+     */
     @PublicAPI(usage = ACCESS)
     public static ArchCondition<JavaClass> beMemberClasses() {
         return BE_MEMBER_CLASSES;
     }
 
+    /**
+     * @return negation of {@link #beMemberClasses()}
+     */
     @PublicAPI(usage = ACCESS)
     public static ArchCondition<JavaClass> notBeMemberClasses() {
         return not(BE_MEMBER_CLASSES);
     }
 
+    /**
+     * @return A condition matching classes that are member classes as defined in {@link JavaClass#isInnerClass()} ()}
```

Comment:
Is the trailing `()}` intended?

Reply:
no, I thought I found them all, but apparently Intellij deems itself to be more intelligent than I find myself